release-notes.md

```diff
@@ -1,3 +1,15 @@
+# Rigel-18.20.0
+## Update Unique_Kms_New_Item_Pending_Moderation_2 email event notification templates ##
```

Comment:
the template you update is New_Item_Pending_Moderation, not Unique_Kms_New_Item_Pending_Moderation_2